src/main/java/duke/main/Storage.java

```diff
@@ -0,0 +1,124 @@
+package duke.main;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.nio.file.FileSystems;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Scanner;
+
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+/**
+ * The class that saves and loads data.
+ */
+public class Storage {
+    private final String filepath;
+
+    /**
+     * Initialisation of the Storage class which requires a filepath to save and load data from.
+     *
+     * @param filepath The relative filepath from where Duke was ran, to save and load data from
+     */
```

Comment:
Should the first word of the Javadoc be a verb instead of a noun? Also, I believe that the first word after `@param` should have the first letter capitalised. :)

Reply:
Sorry I believe that for variables it should be in lowerCamelCase format

Reply:
I read the javadoc wrongly, sorry about my mistake!